terraform/templates/validator.yaml

```diff
@@ -1,5 +0,0 @@
-base:
-  chain_id: "${chain_id}"
-
-validator_network:
-  mutual_authentication: true
```

Comment:
is this the default for validator network?

Reply:
https://github.com/libra/libra/blob/master/config/src/generator.rs#L38